diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter1/IdentifierNamingWarnTest.kt

```diff
@@ -503,6 +503,36 @@ class IdentifierNamingWarnTest : LintTestBase(::IdentifierNaming) {
         lintMethod(code, LintError(1, 7, ruleId, "${BACKTICKS_PROHIBITED.warnText()} `my class name`"))
     }
 
+    @Test
+    @Tag(WarningNames.BACKTICKS_PROHIBITED)
+    fun `should not trigger on underscore`() {
+        val code =
+                """
+                    class SomeClass {
+                        fun bar() {
+                            val ast = list.map {(first, _) -> foo(first)}
+                            
+                            var meanValue: Int = 0
+                                for ((
+                                    _,
+                                    _,
+                                    year, // trailing comma
```

Comment:
Is this example with trailing comma really working? I would assume that parser in 1.3.x will mark it as an error...

Reply:
I took this example from https://kotlinlang.org/docs/reference/coding-conventions.html

Reply:
Yeah, but it's supported since version 1.4.0, and we are currently using 1.3.72

Reply:
Yes, it's not working in 1.3.x